plugins/header_rewrite/parser.h

```diff
@@ -66,7 +66,7 @@ class Parser
   }
 
   bool
-  mod_exist(const std::string m) const
+  mod_exist(const std::string &m) const
   {
     return (std::find(_mods.begin(), _mods.end(), m) != _mods.end());
```

Comment:
What are those parentheses for? RETURN IS NOT A FUNCTION.

Reply:
I know, I didn't write this.